servicetalk-http-api/src/main/java/io/servicetalk/http/api/StreamingHttpResponse.java

```diff
@@ -139,10 +167,12 @@
     /**
      * Returns a {@link StreamingHttpResponse} with its underlying payload transformed to {@link Object}s,
      * with access to the trailers.
+     * @deprecated use {@link #transform(TrailersTransformer)}.
      * @param trailersTransformer {@link TrailersTransformer} to use for this transform.
      * @param <T> The type of state used during the transformation.
      * @return {@code this}
      */
+    @Deprecated
     <T> StreamingHttpResponse transformRaw(TrailersTransformer<T, Object> trailersTransformer);
```

Comment:
This method is still used in `ContentHeadersTest`, please update the test.

Reply:
Lets discuss on https://github.com/apple/servicetalk/pull/1332#discussion_r565782546 due to overlapping discussion.